README.md

```diff
@@ -1,2 +1,87 @@
+
+## Accountability
+
+1. Much like _Core Team_, minutes and reports from the _Dev Team_, and its various sub-groups will be available to view by the general public. Sensitive data, such as budgeting, may be exempt from this.
+2. The _Dev Team_ will submit regular progress reports to the Trustees.
+3. As well as regular volunteers, the _Dev Team_ may contain trustees and members of the _Core Team_, however volunteers should bear in mind any potential conflicts of interest that may arise as a result of this.
+4. Members of the _Dev Team_ should abstain from financial decisions where they are directly involved with the _project group_ of concern. In an event when all members of the _Dev Team_ have to abstain, the decision should be passed to the _Core Team_ or Trustees.
+5. The _Dev Team_ is responsible for the oversight of the various project groups within Student Robotics. The _project groups_ should report their progress regularly to the Dev Team.
+
+## Relationship to the Core Team
+
+- The _Dev Team_ is independent from the _Core Team_, but will work closely with them in certain situations.
+- The _Core Team_ will also communicate priorities for projects to be completed by the next competition cycle. In the event of a Kit Deployment, the _Core Team_ is responsible for budgeting, manufacture, deployment and testing.
+
+## Getting Things Done
+
+For new projects, there is a process to follow to ensure they receive sufficient review and testing before they are implemented. This is shown in the examples below.
+
+For each _project group_ it is recommended that a “Project” is opened and actively maintained with high-level progress updates in the `srobo` GitHub - even the “nice to haves” - if there are good ideas being worked on which get parked here, at least there will be some documentation to pick up on.
+
+Examples of what may be in scope for SR2020: Python 3, new brain board, new IDE.
+
+### Project of significant impact: New brain board
+
+- A (group of) volunteer(s) decide(s) they’d like to work on finding a new single board computer which can serve as the brain board, and port/upgrade software to work on it.
+- New / Updated ticket in srobo/dev-tasks.
+- The volunteers develop a brief specification, including information like:
+    - Functional requirements (e.g. compatible with the rest of the kit, can be powered from Power Board, runs our API, camera isn’t made slower, form factor)
+    - Scope (e.g. excludes any changes to modular kit or API, independent of Python 3 project)
+    - Deliverables (e.g. Board, Software, Documentation for students and volunteers)
+    - Testing requirements (e.g. is demonstrated to run a defined piece of software on a test robot without fault, with a certain response time for libkoki markers, etc.)
+- The _Dev Team_ may also define some limits, such as:
+    - Cut-off date for deployment at SR2020
+    - Cost per unit limit
+    - Evidence shown that more than one board has been considered
```

Comment:
What do you mean by this?

Reply:
evidence that multiple different solutions to the problem have been considered.

Reply:
> multiple different solutions to the problem have been considered.

What if the problem is `pythonic` and so there exists only one obvious solution?

Reply:
> What if the problem is `pythonic` and so there exists only one obvious solution?

## Situation A: This is known when the _Dev Team_ define the requirements

Providing evidence that more than more solution has been considered will not be included in the requirements as the _Dev Team_ are aware that this is not possible.

## Situation B: This is not known when the _Dev Team_ define the requirements

The _project group_ will inform the core team with justification and request for the requirement to be approved. The _Dev Team_ will then consider if the justification is sufficient and then remove the requirement if it is deemed so.

Reply:
In theory Situation B should not be possible if the sole solution is sufficiently obvious, as its obviousness would preclude it being unknown.

Reply:
Thence the vindication should be contemplated as sufficiently satisfactory.

Reply:
I think this is infact fine. The wording above suggests that the dev team _may_ define these limits. To me that is also worded such that the project groups can contest said limits, which accounts for both of the above solutions.